core/oWebSocketServer.php

```
@@ -71,7 +70,7 @@ public function __construct($params)
 		//	3.	establish connection or abort on error
 		$listenstr = $protocol . "://" . $this->host . ":" . $this->port;
 		$this->console("Binding to " . $this->host . ":" . $this->port . " over " . $protocol . "\n");
-		$this->socket = @stream_socket_server($listenstr, $errno, $errstr, STREAM_SERVER_BIND | STREAM_SERVER_LISTEN, $context);
+		$this->socket = stream_socket_server($listenstr, $errno, $errstr, STREAM_SERVER_BIND | STREAM_SERVER_LISTEN, $context);
```

Breaking change.

---

Not that it wont break when we hit php 8.  fail fast I guess :+1:

---

all the "@" changes may not be wrapped in a try catch :thinking:

---

See response on upper comment :)